packages/ui/src/workflows/store/handler.ts

```diff
@@ -74,11 +74,16 @@ export const createState = (
 
 export function createActions(state: HandlerState, client: Client) {
   const actions = {
-    async sendEvent(event: WorkflowEvent, step?: string) {
+    async sendEvent(
+      event: WorkflowEvent | EventEnvelopeWithMetadata,
```

Comment:
templates have a start event that's just a type and value. Support sending that way as well rather than constructing the class